### PR TITLE
chore(android): update android publishing for 5.0 and Gradle 8

### PR DESCRIPTION
**Pull request by carlpoole**

### CODE REVIEW

**Comment by Steven0351** [comment]
Does this also need the `artifact("$buildDir/outputs/aar/${project.getName()}-release.aar")` update instead of using the `components` reference like capacitor did?

File: scripts/android/publish-module.gradle [hunk 1]
```diff
@@ -33,7 +34,7 @@ afterEvaluate {
 
                 // Two artifacts, the `aar` (or `jar`) and the sources
                 if (project.plugins.findPlugin("com.android.library")) {
-                    from components.release
+                    artifact("$buildDir/outputs/aar/${project.getName()}-release.aar")
```
Comment (carlpoole) [inline scripts/android/publish-module.gradle:37]: @Steven0351 Indeed :)

Comment (Steven0351) [reply scripts/android/publish-module.gradle:37]: I totally missed this and I was even actively looking for it 😆

**Review by Steven0351 — APPROVED** [review]

File: scripts/android/publish-module.gradle (hunk 1 above)
Comment (carlpoole) [reply scripts/android/publish-module.gradle:37]: I blame the way Github ordered all the changes, although its just A-Z dir name. But still

**Review by IT-MikeS — APPROVED** [review]

**Pull request merged**